core/controllers/admin_test.py

```diff
@@ -1739,3 +1739,63 @@ def test_check_community_reviewer_rights_without_username(self):
 
         self.assertEqual(response['error'], 'Missing username param')
         self.logout()
+
+
+class IntereactionByExplorationIdHandlerTests(test_utils.GenericTestBase):
+    """Tests for interaction by exploration handler."""
+
+    EXP_ID = 'exp'
+
+    def setUp(self):
+        """Complete the signup process for self.ADMIN_EMAIL."""
+        super(IntereactionByExplorationIdHandlerTests, self).setUp()
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+        self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
+        self.editor_id = self.get_user_id_from_email(self.EDITOR_EMAIL)
+        self.exploration = self.save_new_valid_exploration(
+            self.EXP_ID, self.editor_id, end_state_name='End')
+
+    def test_interaction_by_exploration_id_handler(self):
```

Comment:
Test names should have both the conditions being tested and the expected outcome from the test. In this case the action is included, but conditions & outcome may not be clear. An alternative might be:

```
test_get_interaction_ids_for_valid_exp_id_provides_list_of_ids
```

It's nice to use the following format in general: _test\_action\_with\_condition\_has\_outcome_.

Reply:
done